pkgs/applications/graphics/gnome-obfuscate/default.nix

```diff
@@ -1,73 +1,67 @@
 { stdenv
 , lib
 , fetchFromGitLab
-
 , gettext
 , meson
 , ninja
 , pkg-config
-, python3
 , rustPlatform
 , wrapGAppsHook4
-
```

Comment:
Could you leave the empty lines? I sometimes use them to separate buildinputs/nativebuildinputs/other

Reply:
Nevermind, I see you moved some inputs to their correct place, no need to shuffle arguments around then